src/config/nunjucks.js

```diff
@@ -1,13 +1,22 @@
 const nunjucks = require("nunjucks");
 const i18next = require("i18next");
 
+
+
 module.exports = {
-  configureNunjucks: (app, viewsPath) => {
+  configureNunjucks: (app, viewsPath, url) => {
     const nunjucksEnv = nunjucks.configure(viewsPath, {
       autoescape: true,
       express: app,
       noCache: true
     });
+
+    nunjucksEnv.addGlobal("addLanguageParam", function (language) {
```

Comment:
This function needs to be stored somewhere within the language toggle package.
Because if it's not the case,  it will be duplicated everytime a frontend repo wants to implement the language toggle.